CONTRIBUTING.md

```diff
@@ -1,190 +1,198 @@
 
 1.  Look at the diffs for each file on github, if it is pretty obvious what they are doing is correct then that is a good sign.
 2.  Look at the tests provided with the PR and try to think if there are additional tests that would provide better coverage. If you think additional tests are needed then you should let the owner of the PR know and have them add more tests. All new PRs that are adding new features should be using the new unittest framework, not the old numbered framework.
-3.  Pull the the PR down to your local machine and run all the tests locally.  (Unless it is really trivial)
-4.  If the code looked a bit more complicated when you examined the diffs, then you should bring it up in your editor and look over the code in context and try to understand it better.  If there are issues with how the code is written or if it is unclear about why something was done, then have that conversation with the owner of the PR.
+3.  Pull the the PR down to your local machine and run all the tests locally. (Unless it is really trivial)
+4.  If the code looked a bit more complicated when you examined the diffs, then you should bring it up in your editor and look over the code in context and try to understand it better. If there are issues with how the code is written or if it is unclear about why something was done, then have that conversation with the owner of the PR.
 5.  Many of us have our own projects that exercise Skulpt to its limits. If you are particularly concerned about a PR then you may want to try out the built js files in your environment.
 6.  It is always appropriate to raise questions and have a group conversation about anything that looks particularly problematic or risky.
-7.  With the new style unit tests, You should ask the submitter to file issues for tests that they comment out.  This will let us track completeness over time.  Not every test needs its own issue.  Something like 'when blah feature is added enable  tests x,y,z in foo_test.py' should work.
+7.  With the new style unit tests, You should ask the submitter to file issues for tests that they comment out. This will let us track completeness over time. Not every test needs its own issue. Something like 'when blah feature is added enable tests x,y,z in foo_test.py' should work.
 
 The current group of committers is as follows:
 
-* [Brad Miller](https://github.com/bnmnetp)
-* [Scott Rixner](https://github.com/ixner)
-* [Albert-Jan](https://github.com/albertjan)
-* [Meredydd Luff](https://github.com/meredydd)
-* [Leszek Swirski](https://github.com/LeszekSwirski)
-* [Ben Wheeler](https://github.com/bzwheeler)
+-   [Brad Miller](https://github.com/bnmnetp)
+-   [Scott Rixner](https://github.com/ixner)
+-   [Albert-Jan](https://github.com/albertjan)
+-   [Meredydd Luff](https://github.com/meredydd)
+-   [Leszek Swirski](https://github.com/LeszekSwirski)
+-   [Stu](https://github.com/s-cork)
```

Comment:
```suggestion
-   [Stu Cork](https://github.com/s-cork)
```

Reply:
Thanks, I wasn't sure that Cork was your last name.